UndeclaredThrowableException thrown by reloadChart which leads to LoadChartFiller.fillChart

ERROR [23-January 14:51:08] [https-jsse-nio-8443-exec-1] org.zkoss.ganttz.data.GanttDiagramGraph  - error executing execution listener
java.lang.reflect.UndeclaredThrowableException
        at com.sun.proxy.$Proxy226.reloadChart(Unknown Source)
        at org.libreplan.web.planner.order.OrderPlanningModel$ChangeHooker.lambda$hookIntoGraphChanged$1(OrderPlanningModel.java:631)
        at org.zkoss.ganttz.data.GanttDiagramGraph$2.executeGraphChangeListeners(GanttDiagramGraph.java:310)
        at org.zkoss.ganttz.data.GanttDiagramGraph$2.postAction(GanttDiagramGraph.java:299)
        at org.zkoss.ganttz.util.PreAndPostNotReentrantActionsWrapper.executeWithPreAndPostActions(PreAndPostNotReentrantActionsWrapper.java:55)
        at org.zkoss.ganttz.util.PreAndPostNotReentrantActionsWrapper.doAction(PreAndPostNotReentrantActionsWrapper.java:44)
        at org.zkoss.ganttz.data.GanttDiagramGraph$DependenciesEnforcer.onNewEntrance(GanttDiagramGraph.java:970)
        at org.zkoss.ganttz.data.GanttDiagramGraph$DependenciesEnforcer.access$1400(GanttDiagramGraph.java:770)
        at org.zkoss.ganttz.data.GanttDiagramGraph$DependenciesEnforcer$6.ifNewEntrance(GanttDiagramGraph.java:960)
        at org.zkoss.ganttz.util.ReentranceGuard.entranceRequested(ReentranceGuard.java:60)
        at org.zkoss.ganttz.data.GanttDiagramGraph$DependenciesEnforcer.executeWithPreAndPostActionsOnlyIfNewEntrance(GanttDiagramGraph.java:951)
        at org.zkoss.ganttz.data.GanttDiagramGraph$DependenciesEnforcer.taskPositionModified(GanttDiagramGraph.java:1026)
        at org.zkoss.ganttz.data.GanttDiagramGraph$DependenciesEnforcer.access$1200(GanttDiagramGraph.java:770)
        at org.zkoss.ganttz.data.GanttDiagramGraph.enforceRestrictions(GanttDiagramGraph.java:2040)
        at org.zkoss.ganttz.data.GanttDiagramGraph.add(GanttDiagramGraph.java:2035)
        at org.zkoss.ganttz.data.GanttDiagramGraph.add(GanttDiagramGraph.java:2018)
        at org.zkoss.ganttz.FunctionalityExposedForExtensions.addDependency(FunctionalityExposedForExtensions.java:367)
        at org.zkoss.ganttz.TaskList.addDependency(TaskList.java:341)
        at org.zkoss.ganttz.TaskComponent.doAddDependency(TaskComponent.java:400)
        at org.zkoss.ganttz.TaskComponent$3.service(TaskComponent.java:191)
        at org.zkoss.zk.ui.impl.DesktopImpl.service(DesktopImpl.java:712)
        at org.zkoss.zk.ui.impl.UiEngineImpl.execUpdate(UiEngineImpl.java:1310)
        at org.zkoss.zk.au.http.DHtmlUpdateServlet.process(DHtmlUpdateServlet.java:611)
        at org.zkoss.zk.au.http.DHtmlUpdateServlet.doGet(DHtmlUpdateServlet.java:485)
        at org.zkoss.zk.au.http.DHtmlUpdateServlet.doPost(DHtmlUpdateServlet.java:494)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:648)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:729)
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:230)
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:165)
        at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:192)
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:165)
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:317)
        at org.springframework.security.web.access.intercept.FilterSecurityInterceptor.invoke(FilterSecurityInterceptor.java:127)
        at org.springframework.security.web.access.intercept.FilterSecurityInterceptor.doFilter(FilterSecurityInterceptor.java:91)
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:331)
        at org.springframework.security.web.access.ExceptionTranslationFilter.doFilter(ExceptionTranslationFilter.java:115)
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:331)
        at org.springframework.security.web.session.SessionManagementFilter.doFilter(SessionManagementFilter.java:137)
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:331)
        at org.springframework.security.web.authentication.AnonymousAuthenticationFilter.doFilter(AnonymousAuthenticationFilter.java:112)
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:331)
        at org.springframework.security.web.authentication.rememberme.RememberMeAuthenticationFilter.doFilter(RememberMeAuthenticationFilter.java:158)
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:331)
        at org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter.doFilter(SecurityContextHolderAwareRequestFilter.java:169)
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:331)
        at org.springframework.security.web.savedrequest.RequestCacheAwareFilter.doFilter(RequestCacheAwareFilter.java:63)
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:331)
        at org.springframework.security.web.authentication.www.BasicAuthenticationFilter.doFilterInternal(BasicAuthenticationFilter.java:158)
        at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:331)
        at org.springframework.security.web.authentication.AbstractAuthenticationProcessingFilter.doFilter(AbstractAuthenticationProcessingFilter.java:206)
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:331)
        at org.springframework.security.web.authentication.logout.LogoutFilter.doFilter(LogoutFilter.java:121)
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:331)
        at org.springframework.security.web.header.HeaderWriterFilter.doFilterInternal(HeaderWriterFilter.java:66)
        at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:331)
        at org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter.doFilterInternal(WebAsyncManagerIntegrationFilter.java:56)
        at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:331)
        at org.springframework.security.web.context.SecurityContextPersistenceFilter.doFilter(SecurityContextPersistenceFilter.java:106)
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:331)
        at org.springframework.security.web.FilterChainProxy.doFilterInternal(FilterChainProxy.java:214)
        at org.springframework.security.web.FilterChainProxy.doFilter(FilterChainProxy.java:177)
        at org.springframework.web.filter.DelegatingFilterProxy.invokeDelegate(DelegatingFilterProxy.java:346)
        at org.springframework.web.filter.DelegatingFilterProxy.doFilter(DelegatingFilterProxy.java:262)
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:192)
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:165)
        at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:199)
        at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:96)
        at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:474)
        at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:140)
        at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:79)
        at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:624)
        at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:87)
        at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:349)
        at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:783)
        at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:66)
        at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:745)
        at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1437)
        at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
        at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
        at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.reflect.InvocationTargetException
        at sun.reflect.GeneratedMethodAccessor361.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.libreplan.business.common.AdHocTransactionService.lambda$createOnTransaction$1(AdHocTransactionService.java:99)
        at org.libreplan.business.common.AdHocTransactionService.runOnReadOnlyTransaction(AdHocTransactionService.java:115)
        at sun.reflect.GeneratedMethodAccessor122.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:302)
        at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:190)
        at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
        at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99)
        at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:281)
        at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96)
        at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
        at org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:208)
        at com.sun.proxy.$Proxy90.runOnReadOnlyTransaction(Unknown Source)
        at org.libreplan.business.common.AdHocTransactionService.lambda$createHandler$0(AdHocTransactionService.java:85)
        ... 86 more
Caused by: java.lang.NullPointerException
        at org.zkoss.zk.ui.AbstractComponent$ChildIter.next(AbstractComponent.java:3159)
        at org.zkoss.zk.ui.AbstractComponent$ChildIter.next(AbstractComponent.java:3128)
        at java.util.AbstractList.removeRange(AbstractList.java:570)
        at java.util.AbstractList.clear(AbstractList.java:234)
        at org.libreplan.web.planner.chart.LoadChartFiller.fillChart(LoadChartFiller.java:41)
        at org.libreplan.web.planner.chart.Chart.fillChart(Chart.java:49)
        at org.libreplan.web.planner.order.OrderPlanningModel.lambda$refillLoadChartWhenNeeded$11(OrderPlanningModel.java:898)
        ... 104 more
ERROR [23-January 14:51:16] [https-jsse-nio-8443-exec-9] org.zkoss.ganttz.data.GanttDiagramGraph  - error executing execution listener
java.lang.reflect.UndeclaredThrowableException
        at com.sun.proxy.$Proxy226.reloadChart(Unknown Source)
        at org.libreplan.web.planner.order.OrderPlanningModel$ChangeHooker.lambda$hookIntoGraphChanged$1(OrderPlanningModel.java:631)
        at org.zkoss.ganttz.data.GanttDiagramGraph$2.executeGraphChangeListeners(GanttDiagramGraph.java:310)
        at org.zkoss.ganttz.data.GanttDiagramGraph$2.postAction(GanttDiagramGraph.java:299)
        at org.zkoss.ganttz.util.PreAndPostNotReentrantActionsWrapper.executeWithPreAndPostActions(PreAndPostNotReentrantActionsWrapper.java:55)
        at org.zkoss.ganttz.util.PreAndPostNotReentrantActionsWrapper.doAction(PreAndPostNotReentrantActionsWrapper.java:44)
        at org.zkoss.ganttz.data.GanttDiagramGraph$DependenciesEnforcer.onNewEntrance(GanttDiagramGraph.java:970)
        at org.zkoss.ganttz.data.GanttDiagramGraph$DependenciesEnforcer.access$1400(GanttDiagramGraph.java:770)
        at org.zkoss.ganttz.data.GanttDiagramGraph$DependenciesEnforcer$6.ifNewEntrance(GanttDiagramGraph.java:960)
        at org.zkoss.ganttz.util.ReentranceGuard.entranceRequested(ReentranceGuard.java:60)
        at org.zkoss.ganttz.data.GanttDiagramGraph$DependenciesEnforcer.executeWithPreAndPostActionsOnlyIfNewEntrance(GanttDiagramGraph.java:951)
        at org.zkoss.ganttz.data.GanttDiagramGraph$DependenciesEnforcer.taskPositionModified(GanttDiagramGraph.java:1026)
        at org.zkoss.ganttz.data.GanttDiagramGraph$DependenciesEnforcer.access$1200(GanttDiagramGraph.java:770)
        at org.zkoss.ganttz.data.GanttDiagramGraph.enforceRestrictions(GanttDiagramGraph.java:2040)
        at org.zkoss.ganttz.data.GanttDiagramGraph.add(GanttDiagramGraph.java:2035)
        at org.zkoss.ganttz.data.GanttDiagramGraph.add(GanttDiagramGraph.java:2018)
        at org.zkoss.ganttz.FunctionalityExposedForExtensions.addDependency(FunctionalityExposedForExtensions.java:367)
        at org.zkoss.ganttz.TaskList.addDependency(TaskList.java:341)
        at org.zkoss.ganttz.TaskComponent.doAddDependency(TaskComponent.java:400)
        at org.zkoss.ganttz.TaskComponent$3.service(TaskComponent.java:191)
        at org.zkoss.zk.ui.impl.DesktopImpl.service(DesktopImpl.java:712)
        at org.zkoss.zk.ui.impl.UiEngineImpl.execUpdate(UiEngineImpl.java:1310)
        at org.zkoss.zk.au.http.DHtmlUpdateServlet.process(DHtmlUpdateServlet.java:611)
        at org.zkoss.zk.au.http.DHtmlUpdateServlet.doGet(DHtmlUpdateServlet.java:485)
        at org.zkoss.zk.au.http.DHtmlUpdateServlet.doPost(DHtmlUpdateServlet.java:494)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:648)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:729)
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:230)
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:165)
        at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:192)
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:165)
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:317)
        at org.springframework.security.web.access.intercept.FilterSecurityInterceptor.invoke(FilterSecurityInterceptor.java:127)
        at org.springframework.security.web.access.intercept.FilterSecurityInterceptor.doFilter(FilterSecurityInterceptor.java:91)
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:331)
        at org.springframework.security.web.access.ExceptionTranslationFilter.doFilter(ExceptionTranslationFilter.java:115)
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:331)
        at org.springframework.security.web.session.SessionManagementFilter.doFilter(SessionManagementFilter.java:137)
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:331)
        at org.springframework.security.web.authentication.AnonymousAuthenticationFilter.doFilter(AnonymousAuthenticationFilter.java:112)
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:331)
        at org.springframework.security.web.authentication.rememberme.RememberMeAuthenticationFilter.doFilter(RememberMeAuthenticationFilter.java:158)
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:331)
        at org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter.doFilter(SecurityContextHolderAwareRequestFilter.java:169)
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:331)
        at org.springframework.security.web.savedrequest.RequestCacheAwareFilter.doFilter(RequestCacheAwareFilter.java:63)
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:331)
        at org.springframework.security.web.authentication.www.BasicAuthenticationFilter.doFilterInternal(BasicAuthenticationFilter.java:158)
        at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:331)
        at org.springframework.security.web.authentication.AbstractAuthenticationProcessingFilter.doFilter(AbstractAuthenticationProcessingFilter.java:206)
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:331)
        at org.springframework.security.web.authentication.logout.LogoutFilter.doFilter(LogoutFilter.java:121)
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:331)
        at org.springframework.security.web.header.HeaderWriterFilter.doFilterInternal(HeaderWriterFilter.java:66)
        at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:331)
        at org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter.doFilterInternal(WebAsyncManagerIntegrationFilter.java:56)
        at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:331)
        at org.springframework.security.web.context.SecurityContextPersistenceFilter.doFilter(SecurityContextPersistenceFilter.java:106)
        at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:331)
        at org.springframework.security.web.FilterChainProxy.doFilterInternal(FilterChainProxy.java:214)
        at org.springframework.security.web.FilterChainProxy.doFilter(FilterChainProxy.java:177)
        at org.springframework.web.filter.DelegatingFilterProxy.invokeDelegate(DelegatingFilterProxy.java:346)
        at org.springframework.web.filter.DelegatingFilterProxy.doFilter(DelegatingFilterProxy.java:262)
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:192)
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:165)
        at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:199)
        at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:96)
        at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:474)
        at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:140)
        at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:79)
        at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:624)
        at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:87)
        at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:349)
        at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:783)
        at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:66)
        at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:745)
        at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1437)
        at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
        at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
        at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.reflect.InvocationTargetException
        at sun.reflect.GeneratedMethodAccessor361.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.libreplan.business.common.AdHocTransactionService.lambda$createOnTransaction$1(AdHocTransactionService.java:99)
        at org.libreplan.business.common.AdHocTransactionService.runOnReadOnlyTransaction(AdHocTransactionService.java:115)
        at sun.reflect.GeneratedMethodAccessor122.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:302)
        at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:190)
        at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
        at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99)
        at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:281)
        at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96)
        at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
        at org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:208)
        at com.sun.proxy.$Proxy90.runOnReadOnlyTransaction(Unknown Source)
        at org.libreplan.business.common.AdHocTransactionService.lambda$createHandler$0(AdHocTransactionService.java:85)
        ... 86 more
Caused by: java.lang.NullPointerException
        at org.zkoss.zk.ui.AbstractComponent$ChildIter.next(AbstractComponent.java:3159)
        at org.zkoss.zk.ui.AbstractComponent$ChildIter.next(AbstractComponent.java:3128)
        at java.util.AbstractList.removeRange(AbstractList.java:570)
        at java.util.AbstractList.clear(AbstractList.java:234)
        at org.libreplan.web.planner.chart.LoadChartFiller.fillChart(LoadChartFiller.java:41)
        at org.libreplan.web.planner.chart.Chart.fillChart(Chart.java:49)
        at org.libreplan.web.planner.order.OrderPlanningModel.lambda$refillLoadChartWhenNeeded$11(OrderPlanningModel.java:898)
        ... 104 morePress any key to continue...

--- a/libreplan-webapp/src/main/java/org/libreplan/web/planner/chart/LoadChartFiller.java
+++ b/libreplan-webapp/src/main/java/org/libreplan/web/planner/chart/LoadChartFiller.java
@@ -38,7 +38,9 @@ public abstract class LoadChartFiller extends ChartFiller {
 
     @Override
     public void fillChart(Timeplot chart, Interval interval, Integer size) {
-        chart.getChildren().clear();
+        if (chart.getChildren() != null) {
+            chart.getChildren().clear();
+        }
         chart.invalidate();
 
         if (getOptionalJavascriptCall() != null) {

